tools/dailyjobs_query.py

```diff
@@ -34,13 +39,19 @@ def parse_args(argv=None):
 
 
 def updatedb(date, platform, branch, numpushes, numjobs, sumduration):
-    cur.execute('delete from dailyjobs where date="%s" and branch="%s" and platform="%s"'
-                % (date, branch, platform))
-    query = 'insert into dailyjobs (date, platform, branch, numpushes, numjobs, sumduration) \
-             values ("{0}", "{1}", "{2}", {3}, {4}, {5})'
-    query = query.format(date, platform, branch, numpushes, numjobs, sumduration)
-    print query
-    cur.execute(query)
+
+    dailyjob = Dailyjobs(date, platform, branch, numpushes, numjobs, sumduration)
+    try:
+        session.add(dailyjob)
+    except Exception as e:
+        logger.warning(e)
+        session.rollback()
+    message = 'insert into dailyjobs (date, platform, branch, numpushes, numjobs, sumduration) ' \
+              'values ("{0}", "{1}", "{2}", {3}, {4}, {5})'.format(date, platform, branch,
+                                                                   numpushes, numjobs,
+                                                                   sumduration)
+    print message
```

Comment:
we print here, and it looks like we are not deleting the old stuff?  is there a reason to not 'delete from dailyjobs where ...' ?